javascript-package/src/parameters.js

```diff
@@ -0,0 +1,156 @@
+import { camelCase } from "change-case";
+
+const parameterize = (parametersArray) => {
+  return new Set(parametersArray.map(camelCase));
+};
+
+export const playersXgoalsParameters = parameterize([
```

Comment:
This may not be feasible because of the need to convert to camel case, but on the R/Python side, we're treating these as keyword arguments and letting the API dictate what's a valid request.

(For what it's worth, the response message in the case of a bad argument may not be the most verbose at the moment, but that's something we'll eventually get around to enhancing.)

Reply:
I added the validation because the api returns a 500 when you send bad parameters, but doesn't tell you _which_ parameters are the problem. I thought this would make it easier to debug. Also, I am only emitting warnings, and then passing through the parameters to the fetch request after re-casing them, so you could hypothetically continue to use the "unsupported" parameters, just with warnings fwiw.